packages/round-manager/src/features/round/__tests__/ViewRoundPage.test.tsx

```diff
@@ -312,8 +312,233 @@ describe("View Round", () => {
+  //       data: [makeQFDistribution(), makeQFDistribution()],
+  //       error: null,
+  //       loading: false,
+  //     },
+  //   }));
+  //
+  //   // mock file.arrayBuffer
+  //   const mockFile = new File([""], "test.json", { type: "application/json" });
+  //   const mockFileArrayBuffer = jest.fn().mockResolvedValue(new ArrayBuffer(0));
+  //   Object.defineProperty(mockFile, "arrayBuffer", {
+  //     value: mockFileArrayBuffer,
+  //   });
+  //
+  //   // mock the text decoder
+  //   const mockTextDecoder = jest.fn().mockReturnValue({
+  //     decode: jest.fn().mockReturnValue(`
+  //     [
+  //       {
+  //         "bad":"bad",
+  //         "bad":"bad-0xebdb4156203c8b35b7a7c6f320786b98e5ac67c3",
+  //         "bad":6202,
+  //         "bad":0.0560505703204296
+  //       }
+  //     ]
+  //     `),
+  //   });
+  //   Object.defineProperty(window, "TextDecoder", {
+  //     value: mockTextDecoder,
+  //   });
+  //
+  //   const roundEndTime = faker.date.recent();
+  //   const roundStartTime = faker.date.past(1, roundEndTime);
+  //   const applicationsEndTime = faker.date.past(1, roundStartTime);
+  //   const applicationsStartTime = faker.date.past(1, applicationsEndTime);
+  //
+  //   const approvedProjects = [
+  //     makeApprovedProjectData(),
+  //     makeApprovedProjectData(),
+  //     makeApprovedProjectData(),
+  //   ];
+  //   mockRoundData = makeRoundData({
+  //     applicationsStartTime,
+  //     applicationsEndTime,
+  //     roundStartTime,
+  //     roundEndTime,
+  //     approvedProjects,
+  //   });
+  //   render(
+  //     wrapWithBulkUpdateGrantApplicationContext(
+  //       wrapWithApplicationContext(
+  //         wrapWithReadProgramContext(
+  //           wrapWithRoundContext(<ViewRoundPage />, {
+  //             data: [mockRoundData],
+  //             fetchRoundStatus: ProgressStatus.IS_SUCCESS,
+  //           })
+  //         )
+  //       )
+  //     )
+  //   );
+  //   const fundingAdminTab = screen.getByTestId("funding-admin");
+  //   fireEvent.click(fundingAdminTab);
+  //   expect(screen.getByTestId("match-stats-title")).toBeInTheDocument();
+  //   expect(screen.getByTestId("matching-stats-table")).toBeInTheDocument();
+  //   expect(screen.getByTestId("custom-or-default-test-id")).toBeInTheDocument();
+  //
+  //   const customRadioButton = screen.getByTestId("custom-radio-test-id");
+  //   fireEvent.click(customRadioButton);
+  //   expect(screen.getByTestId("drop-zone-test-id")).toBeInTheDocument();
+  //
+  //   // test dropzone .json file upload
+  //   const dropzone = screen.getByTestId("drop-zone-test-id");
+  //   fireEvent.drop(dropzone, { dataTransfer: { files: [mockFile] } });
+  //
+  //   expect(mockFile.arrayBuffer).toHaveBeenCalled();
+  //
+  //   // expect the table to not be rendered
+  //   await waitFor(() => {
+  //     expect(
+  //       screen.queryByTestId("matching-stats-table")
+  //     ).not.toBeInTheDocument();
+  //   });
+  // });
```

Comment:
^ shouldn't tests be in ViewAdminFundingPage.test ?

Reply:
They can be - but it is a subcomponent of the view round page. So loading, clicking tabs, for other tests is also in the view round tests.